check-versions/tests/VersionTest.php

```diff
@@ -108,7 +108,7 @@ public function testNginxVersion()
     {
```

Comment:
Oh, can you remove `testNginxVersion` and other nginx related stuff in this file.